lib/Home.dart

```diff
@@ -250,7 +250,7 @@ class Home extends StatelessWidget {
                         Icon(Icons.arrow_forward, size: 16)
                       ]),
                       SizedBox(height: 12),
-                      UpcomingRides(),
+                      Container(height: 225, child: UpcomingRide()),
```

Comment:
You can perhaps use a SingleChildScrollVIew with a horizontal axis alignment to avoid using a hardcoded height. This should most likely be added to the UpcomingRide Widget.